junit/src/test/scala/scala/collection/parallel/TaskTest.scala

```diff
@@ -29,7 +29,12 @@ class TaskTest {
         t
       }
     }
-    def mkPool(name: String) = new ForkJoinPool(1, mkFactory(name), null, false)
+    def mkPool(name: String) = {
+      val parallelism = 1
+      val handler: Thread.UncaughtExceptionHandler = null
+      val asyncMode = false
+      new ForkJoinPool(parallelism, mkFactory(name), handler, asyncMode)
+    }
```

Comment:
Is this because we can't use named arguments for Java-defined methods? 🤔

Reply:
Yes, I believe scalac still complains, although I didn't try again here. Fool me once, as the saying goes.

I actually did this because my first version of the test was this plus throwing on `run`, but that fails outright and hangs!